browser/brave_rewards/android/brave_rewards_native_worker.h

```diff
@@ -254,6 +254,7 @@ class BraveRewardsNativeWorker
   void onPublisherBanner(brave_rewards::mojom::PublisherBannerPtr wallet);
 
   void OnOneTimeTip(brave_rewards::mojom::Result result);
+  void onSendContribution(bool result);
```

Comment:
Nit: casing should match (`On...`). For this signature my suggestion would be something like:

```
void OnContributionSent(bool success);
```